git-secrets

```diff
@@ -69,7 +69,9 @@ scan() {
 }
 
 scan_history() {
-  local to_scan=$(git rev-list --all)
+  local patterns=$(load_patterns | tr ' ' '|')
+  [ -z "${patterns}" ] && return 0
+  local to_scan=$(git log --all -S"${combined_patterns}" --pickaxe-regex --pretty=%H)
   local output=$(git_grep "" "${to_scan}")
```

Comment:
If we already did a regex scan of the logs with `--pickaxe-regex`, then why do we need to run `git_grep` again? Is this to normalize the output to be consistent?